Jenkinsfile

```diff
@@ -0,0 +1,101 @@
+#!groovy
+
+timestamps {
+  ansiColor('xterm') {
+    node('node-small') {
+      stage('SETUP') {
+        deleteDir()
+        dir('scm'){
+          checkout scm
+        }
+        dir('solrHome') {
+          git branch: SOLR_HOME_BRANCH, url: SOLR_HOME_GIT_URL
+        }
+      }
+      stage('BUILD') {
+        dir('scm') {
+          // Build the docker image, push to aws
+          sh """
+            docker build --tag ${DOCKER_REGISTRY_ADDRESS}/${STORE_NAMESPACE}/ep-store:\$(jq -r .version package.json) \
+              --build-arg BUILD_DATE="\$(date --rfc-3339=seconds)" --build-arg VERSION=\$(jq -r .version package.json) \
+              --build-arg VCS_REF=\$(git rev-parse HEAD) -f ./docker/dev/Dockerfile .
+            eval "\$(aws ecr get-login --no-include-email)"
+            docker push ${DOCKER_REGISTRY_ADDRESS}/${STORE_NAMESPACE}/ep-store:\$(jq -r .version package.json)
```

Comment:
it will not update :latest, i'm just reusing this from previous pipeline, ideally we should be tagging them with timestamps. I think we should avoid using :latest though. here is from a thread discussion in #pd (for those wondering why latest should be avoided as an image tag, it’s because the word latest means different things to different people, often built last vs. built using the newest version of the code)